packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -58,12 +58,12 @@ var dasherize = Ember.String.dasherize;
               "data": [
                 {
                   "id": "3",
-                  "type": "players"
+                  "type": "player"
```

Comment:
@bmac is correct. The internal Ember Data JSON API format differs a bit from the one used by `JSONAPISerializer`.